sdk/internal/recording/recording.go

```diff
@@ -512,13 +510,19 @@ func StartRecording(t *testing.T, pathToRecordings string, options *RecordingOpt
 	if err != nil {
 		return err
 	}
-	recordingId = resp.Header.Get(IdHeader)
+	recId := resp.Header.Get(IdHeader)
+	if recId == "" {
+		b, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			return err
+		}
+		return fmt.Errorf("Recording ID was not returned by the response. Response body: %s", b)
```

Comment:
NIT: would it be helpful to say that the specific header was not found? Could aid in diagnostics if someone is trying to figure out what's wrong strictly by the error message.